src/style.css

```
@@ -0,0 +1,94 @@
+html {
+  --main-color: #3d4e61;
+  --box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.search-bar {
+  margin: 20px 9px 0;
+}
+
+.search-bar__input {
+  padding-left: 17px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Move `font-family` into html selector
For input use `inherit` value
```suggestion
  font-family: inherit;
```